packages/core/docs/README.md

```diff
@@ -27,12 +27,6 @@ _Vue Storefront_ is a ___platform-agnostic e-commerce PWA frontend framework___
   <figcaption style="font-size: 0.9rem">(Click to zoom)</figcaption>
 </figure>
 
-That's a mouthful, so let's break it down:
- - __platform-agnostic__ - we made it possible to work with any platform and service you already use, as long as it has an API like REST or GraphQL.
- - __e-commerce__ - today's shops are much more than just products and carts. That's why we made it easy to integrate other types of services, such as helper service for ERP, versatile search features for PIM, portable checkout for 3rd party payment kiosk, and more.
- - __PWA__ - it's the technology of the future, designed to give the best performance on any device, with native-like features to satisfy your customer's needs.
- - __frontend framework__ - _Vue Storefront_ is a set of modular features glued together using _interfaces_ and _factories_ and powered by [Nuxt.js](https://nuxtjs.org/).
-
```

Comment:
(Out of the scope for this PR)
Maybe we can have a glossary page with those terms and their meanings.

Reply:
It's already on our roadmap 😊